guides/release/tutorial/part-2/service-injection.md

```diff
@@ -4,7 +4,7 @@ As promised, we will now work on implementing the share button!
 
 <!-- TODO: make this a gif instead -->
 
-<img src="/images/tutorial/part-2/service-injection/suggested-tweet@2x.png" alt="The working share button by the end of the chapter" width="1024" height="449">
+![The working share button by the end of the chapter](/images/tutorial/part-2/service-injection/suggested-tweet@2x.png)
```

Comment:
Replace this with another image. This screenshot was deleted in https://github.com/ember-learn/super-rentals-tutorial/pull/146

Reply:
Fixed in https://github.com/ember-learn/super-rentals-tutorial/pull/150